### PR TITLE
docs: fix the documentation of source build error

### PR DESCRIPTION
**Pull request by easy1090**
fix the documentation of source build error

### CODE REVIEW

File: test/components/tsconfig.json [hunk 1]
```diff
@@ -12,7 +12,6 @@
     "resolveJsonModule": true,
     "moduleResolution": "Bundler",
     "useDefineForClassFields": true,
-    "composite": true,
```
Comment (chenjiahan) [inline test/components/tsconfig.json:15]: Why delete this

Comment (easy1090) [reply test/components/tsconfig.json:15]: I check the Typescript document，maybe no need composite:true for app project?

https://cspotcode.com/posts/tsconfig-composite-configuration

https://www.typescriptlang.org/docs/handbook/project-references.html

Comment (chenjiahan) [reply test/components/tsconfig.json:15]: `components` is a lib project, not app

Comment (easy1090) [reply test/components/tsconfig.json]: ok, fixed this one.

**Pull request merged**